docs/UI/SetAppIcon/index.md

```diff
@@ -42,7 +42,7 @@ SetAppIcon
 ```json
 {
   "id" : 88,
-  "jsonrpc" : "2.0",
+  "jsonrpc" : "2.
```

Comment:
Mistaken deletion, but it breaks the JSON formatting

Reply:
@jacobkeeler , thank you.
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/95/commits/4b39d1b8620a2725a598945b2dee593e924f6081